docs/scripts.rst

```diff
@@ -353,8 +358,15 @@ Full control over column names and data types
 
 By default the Retriever automatically detects both column names and data types, but you can also exercise complete
 control over the structure of the resulting database by adding column names and types.
+
+It is recommended to describe the schema of the table while creating JSON file. This enables processing of the data faster since column detection increases the processing time.
```

Comment:
"It is recommended to describe the schema of the table while creating **the** JSON file. This enables processing of the data faster since column detection increases the processing time. "